.cbmc-batch/include/make_common_data_structures.h

```diff
@@ -32,6 +32,12 @@ void ensure_md_context_has_allocated_members(struct aws_cryptosdk_md_context *ct
 /* Allocates the members of the context and ensures that internal pointers are pointing to the correct objects. */
 void ensure_sig_ctx_has_allocated_members(struct aws_cryptosdk_sig_ctx *ctx);
 
+bool aws_cryptosdk_edk_list_is_bounded(
```

Comment:
These would be useful for some other proofs, including the ones I did.  Lets make an issue to use them.

Reply:
https://github.com/aws/aws-encryption-sdk-c/issues/542 done.